VisualRiders.PointOfSale.Project/Repositories/DiscountItemsRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using VisualRiders.PointOfSale.Project.Models;
+
+namespace VisualRiders.PointOfSale.Project.Repositories;
+
+public class DiscountItemsRepository : RepositoryBase<DiscountItem>
+{
+    public DiscountItemsRepository(PointOfSaleContext context) : base(context)
+    {
+    }
+
+    public List<DiscountItem>? GetItemsByDiscountId(int id)
```

Comment:
This can never be `null` as it searches for items, matching a specific criteria and returns an empty list if no items match.